recipes-browser/cog/cog.inc

```diff
@@ -9,6 +9,8 @@ BUGTRACKER = "https://github.com/Igalia/cog/issues"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://COPYING;md5=bf1229cd7425b302d60cdb641b0ce5fb"
 
+BBCLASSEXTEND = "native"
```

Comment:
What is the rationale behind to make this recipe native as well?.

Reply:
Recipes which extends native class can be used to build the component for both target and native in a single build.

Reply:
I see your point. What I can not see is the why we could be interested on that. AFAIK the native package are useful when they are hosts dependendencies during the build time of a third package. For cog, what I can't foresee is another package requiring a native cog for building itself. What am I missing?